openwisp_controller/config/base/vpn.py

```diff
@@ -187,10 +203,35 @@ def _validate_subnet_ip(self):
                     {'ip': _('VPN IP address must be within the VPN subnet')}
                 )
 
+    def _validate_host_and_auth_token(self):
+        if self.host and self.auth_token and self._is_backend_type('zerotier'):
+            try:
+                response = ZerotierService(self.host, self.auth_token).get_node_status()
+            except ConnectionError as err:
+                err_msg = f'Failed to connect to the ZeroTier controller, error: {err}'
+                raise ValidationError({'host': _(err_msg)})
+            if response.status_code == 401:
+                err_msg = (
+                    f'Failed to connect to the ZeroTier controller, '
+                    'please ensure you are using the correct authorization token '
+                    f'(error: {response.reason}, status code: {response.status_code})'
```

Comment:
```suggestion
                    f'Authozation failed for ZeroTier controller,'
                    ' ensure you are using the correct authorization token '
                    f'(error: {response.reason})'
```

There is not much benefit to show status_code here. And, if the `response.reason` only says "authorization failed", the we should omit it as well for brevity.

Reply:
Yes, it says `"Unauthorized"` as the error reason. I have removed that from the validation message. Thanks!"
![Screenshot from 2023-07-23 22-21-33](https://github.com/openwisp/openwisp-controller/assets/56113566/235124d8-3d84-4746-91bf-9918450f8822)